client/question/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -53,6 +55,33 @@ tie.factory('PythonPrereqCheckService', [
       });
     };
 
+    /**
+     * Processes studentCode and converts all single quote + double quote str
+     * to filler character "x".
+     * This is to preserve character length for error checks.
+     *
+     * @param {string} [code] The code to convert
+     * @return {string} code where all 'foo' & "bar" converted to xxxxx & xxxxx
+     */
+    var obscureStringCharacters = function(code) {
```

Comment:
getObscuredCode might be better. The issue is that "obscure" could serve as a verb or an adjective, so it's not immediately clear from reading the name whether obscureStringCharacters is a method or a set/list of obscure string chars.

Reply:
Reasoning make sense, updated as getObscuredCode.